pnpm-lock.yaml

```diff
@@ -5769,43 +5797,55 @@ packages:
       - supports-color
     dev: true
 
-  /@mdx-js/loader/0.18.0_uuaxwgga6hqycsez5ok7v2wg4i:
-    resolution: {integrity: sha512-eRgtB14JwyIiZZPXjrpYhSHHQ5+GtZ5cbG744EV2DZVKjxxg4OT/EtKc4JoxWHRK2HVU6W7cf8IXjQpqDviRuQ==}
+  /@mdx-js/loader/1.6.22_react@18.2.0:
+    resolution: {integrity: sha512-9CjGwy595NaxAYp0hF9B/A0lH6C8Rms97e2JS9d3jVUtILn6pT5i5IV965ra3lIWc7Rs1GG1tBdVF7dCowYe6Q==}
     dependencies:
-      '@mdx-js/mdx': 0.18.2_@babel+core@7.18.0
-      '@mdx-js/tag': 0.18.0_react@18.2.0
-      loader-utils: 1.4.0
+      '@mdx-js/mdx': 1.6.22
+      '@mdx-js/react': 1.6.22_react@18.2.0
```

Comment:
Test nownseems passing as mdx loader have mdx/react as transitive dependency. But I do not understand how existing test passed without this dependency until today 🤔